starfish/image/_filter/richardson_lucy_deconvolution.py

```diff
@@ -16,33 +16,57 @@
 
 
 class DeconvolvePSF(FilterAlgorithmBase):
+    """Deconvolve a user-provided isotropic point spread function of size sigma
```

Comment:
Deconvolve a point spread function from the image. The point spread function is assumed to be an isotropic Gaussian, with a user specified standard deviation, sigma. 

It may be worth pointing out that there are currently several issues with this function? See https://github.com/spacetx/starfish/issues/731